ui5/eve7/lib/GlViewerThree.js

```diff
@@ -102,12 +102,13 @@ sap.ui.define([
 
          // Lights are positioned in resetRenderer
 
-         this.point_lights = new THREE.Object3D;
-         this.point_lights.add(new THREE.PointLight(0xffffff, 0.7)); // R
-         this.point_lights.add(new THREE.PointLight(0xffffff, 0.7)); // G
-         this.point_lights.add(new THREE.PointLight(0xffffff, 0.7)); // B
+         this.point_lights = new THREE.Object3D();
+         this.point_lights.add(new THREE.AmbientLight(0xffffff, 1)); // R
+         this.point_lights.add(new THREE.AmbientLight(0xffffff, 1)); // G
+         this.point_lights.add(new THREE.AmbientLight(0xffffff, 1)); // B
```

Comment:
Can you change those three AmbientLights to Directional? Otherwise everything looks flat.